actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -40,11 +40,14 @@ def url_for(options = nil) # :nodoc:
         end
       end
 
+      # Returns previous URL in the browser history from request referrer or JavaScript history back method
       def _back_url # :nodoc:
         _filtered_referrer || 'javascript:history.back()'
       end
       protected :_back_url
 
+      # Returns referrer URL of the current request and also ensures it doesn't contain a JavaScript protocol
+      # Raises an exception if referrer has invalid URI format
```

Comment:
Not sure if this is true, we specifically rescue it, can you show an example where it might raise?
If so, we need a test to cover it, I can't see this behaviour covered.